debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/J9ObjectFieldOffsetIterator_V1.java

```diff
@@ -214,7 +215,20 @@ private void fieldOffsetsFindNext() throws CorruptDataException {
 							J9ClassPointer fieldClass = valueTypeHelper.findJ9ClassInFlattenedClassCacheWithFieldName(instanceClass, J9ROMFieldShapeHelper.getName(localField));
 							if (valueTypeHelper.isJ9FieldIsFlattened(fieldClass, localField)) {
 								UDATA size = null;
-								if (valueTypeHelper.isJ9ClassLargestAlignmentConstraintDouble(fieldClass)) {
+								boolean forceDoubleAlignment;
+								if (fj9object_t_SizeOf == U32.SIZEOF) {
+									UDATA instanceSize = fieldClass.totalInstanceSize();
+									UDATA doubleSize = new UDATA(U64.SIZEOF);
+									if (valueTypeHelper.classRequires4BytePrePadding(fieldClass)) {
+										instanceSize = instanceSize.sub(U32.SIZEOF);
+									}
+									forceDoubleAlignment = modifiers.allBitsIn(J9JavaAccessFlags.J9AccVolatile) && (instanceSize.eq(doubleSize));
```

Comment:
Please remove the redundant parentheses in `(instanceSize.eq(doubleSize)`.
This line is also excessively long (it can't be viewed in its entirety on the web).

Reply:
Fixed.